client/components/AddNew.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import { Form } from 'semantic-ui-react'
+import { Link } from 'react-router-dom'
+
+export default class AddNew extends Component {
+  state = {}
+
+  handleChange = (e, { value }) => this.setState({ value })
```

Comment:
This is a good start, you're on the right track. Remember computed property syntax?
```js
handleChange = e => this.setState({ [ e.target.name ]: e.target.value })
```

Reply:
Added changes to handleChange event